### PR TITLE
Feature/check rules config

### PR DESCRIPTION
**Pull request by fdurand**
## Description

Check if the apache filter and vlan filter syntax are good.
## Impacts

PacketFence will not start if the syntax is wrong.
## Delete branch after merge

YES

### CODE REVIEW

File: NEWS.asciidoc [hunk 1]
```diff
@@ -42,6 +42,7 @@ Enhancements
 * Reduce deadlocks caused by the cleaning of the iplog table
 * Reduce deadlocks caused by the cleaning of the locationlog table
 * Reorganize the portal profile option page
+* Added checkup on apache filters and vlan filters
```
Comment (extrafu) [inline NEWS.asciidoc]: apache -> Apache
vlan -> VLAN

File: lib/pf/vlan/filter.pm [hunk 2]
```diff
@@ -98,8 +98,9 @@ sub dispatchRule {
     my ($self, $rule, $switch, $ifIndex, $mac, $node_info, $connection_type, $user_name, $ssid, $radius_request, $name) = @_;
     my $logger = Log::Log4perl::get_logger( ref($self) );
 
-    if (!defined($rule)) {
-        $logger->error("The rule $name you try to test doesn´t exist");
+    if (!defined($rule) || !(defined($rule->{'filter'}) && defined($rule->{'operator'}) && defined($rule->{'value'}) ) ) {
+        $logger->error("The rule $name you try to test doesn´t exist or an attribute is missing");
```
Comment (extrafu) [inline lib/pf/vlan/filter.pm]: doesn´t -> doesn't

File: lib/pf/pfcmd/checkup.pm [hunk 3]
```diff
@@ -996,6 +999,58 @@ sub portal_profiles {
...
+                if (!defined($ConfigVlanFilters{$rule}->{'operator'}));
+            add_problem ( $FATAL, "Missing value attribute in $rule vlan filter rule")
+                if (!defined($ConfigVlanFilters{$rule}->{'value'}));
+        }
+    }
+}
+
+=item apache_filter_rules
+
+Make sure that the minimum parameters has been defined in apache filter rules
```
Comment (julsemaan) [inline lib/pf/pfcmd/checkup.pm]: *have been defined

File: lib/pf/pfcmd/checkup.pm [hunk 4]
```diff
@@ -996,6 +999,58 @@ sub portal_profiles {
     }
 }
 
+=item vlan_filter_rules
+
+Make sure that the minimum parameters has been defined in vlan filter rules
```
Comment (julsemaan) [inline lib/pf/pfcmd/checkup.pm]: *have been defined

**Comment by julsemaan** [comment]
Tested briefly and it does what it's supposed to do.

Looks good for merge unless someone else wants to review

**Pull request merged**